### PR TITLE
jeandavidt/issue135

### PR DESCRIPTION
**Pull request by jeandavidt**
First stab at fixing #135

I left the Reporter, Lab and AssayMethod rows blank for now, as they are a bit trickier to include than the others. I'm open to ideas on how to include them, or if we don't deem it necessary, then maybe we could leave them out for this release?

### CODE REVIEW

**Comment by DougManuel** [comment]
Can you explain or clarify _column in valueColumnFormula and descColumnFormula?

Is this the value in valueColumns? (i.e. analysisDate, reportDate, qualityFlag, value, notes for WWMeasure)

**Comment by DougManuel** [comment]
This PR is timely because I am working on VOC reporting. We have an online meeting tomorrow. I suggest we keep this PR open until we can discuss VOCs to ensure there are no breaking changes.

A few comments now:
1) Generally, looks excellent. I very much like the format of the Wide table and column headings. 

2) I suggest that you start to create a readme (Wide.md?) for this Wide table that explains or walks through how to generate a wide table name and also explains the headers.

I suggested to @yisheikh to create a readme for the Validate table (Validate.md) that he was going to put in the main directory until we find a better place for it. 

3) I also suggest that we add each header to the ` Vairables.csv`. It may even be that the `tableName` and `tableNickname` columns go into `VariableCategories.csv`. I know this may seem odd and we can discuss. I made the same suggestion to @yisheikh for the `Validate` table. Basically, we need to capture the metadata for this table and `Variables.csv` is where this metadata goes. 

I suggest that it is important, for example, to write an English and French description of the headers (variables) for the `Wide` table. The logical place for that is `Variables.csv`. 

4) I don't quite follow the `joinFormula` column. That would be good to describe in the readme.

Otherwise looks excellent, but let's have that discussion about the VOCs.

**Comment by jeandavidt** [comment]
2. I will write a wide.md with a 'recipe' and clarifications - indeed, the joinFormula needs some explaining

3. I agree that we need to put the column header descriptions somewhere, but I don't think we can just put them in `Variables.csv` without making other changes to it then to the sql creation scripts. I say that because wight now the R script takes what is in `Variables.csv` to generate a SQL statement that creates an instance of the DB, but we don't want Wide.csv to be part of the schema. So either we add a column to `Variables.csv` (e.g., `includeInERD`) to filter out info about the wide table, or we store info about the wide table elsewhere (in the markdown, maybe).

Having the table nicknames in `VariableCategory.csv` is less of a concern because I don't think that that file is implicated in the DB creation script (it is implicated in the metadata.md creation script, but I don't think there would be any ill effect there)

**Comment by DougManuel** [comment]
I agree that we don't want to create an sql database with the wide and validation schema. That just doesn't make sense. We probably want to drop from the ERD. I think we want it on metadata.md but we will want to present wide.csv and validate.csv with different context or explanation on metadata.md.

I like the suggestion of includeInERD, includeInSQL approach and suggest we go with that. We can also have an includeInMetadata -- for metadata.md if we have concerns. I think that should be a fairly straightforward refactor.

File: wide.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+# The ODM Wide Table
+
+The multiple tables of the data model make it easy to store various types of data. Though the ODM structure allows for flexibility and specificity when committing data, this structure is not optimal for consulting and displaying the data, especially when trying to follow the evolution of different quantities over time. To serve this purpose, we propose a systematic template to generate a table that distributes the various recorded time series from a given site into separate columns. The data is spread across rows according to time. We call this table, the "wide" table, since it spreads data horizontally into individual columns.
```
Comment (DougManuel) [inline wide.md:3]: "The 'wide' table has one row for each day; whereas, the regular ODM tables have one row for each measurement."

**Pull request merged**